src/table/utils/__tests__/handle-accessiblity.spec.ts

```diff
@@ -100,7 +107,7 @@ describe('handle-accessibility', () => {
     });
 
     it('should return null', () => {
-      cell = elementCreator('div', '-1');
+      cell = elementCreator('div', '-1') as HTMLTableCellElement;
```

Comment:
HTMLTableCellElement starts with <table> not <div>. so I think it is HTMLDivElement